src/main/java/dev/waterdog/utils/ProxyConfig.java

```diff
@@ -85,6 +85,10 @@
     @Comment("If enabled, only players which are authenticated with XBOX Live can join. If disabled, anyone can connect *with any name*")
     private boolean onlineMode = true;
 
+    @Path("enable_ipv6")
+    @Comment("If enabled, the proxy will be able to bind to an Ipv6 Address")
+    private boolean enableIpv6 = true;
```

Comment:
Some networks may not support ipv6, lets disable it be default.

Reply:
Agreed